README.md

```diff
@@ -4,99 +4,99 @@
 
 [Rspamd](https://rspamd.com) is an advanced spam filtering system that allows evaluation of messages by a number of
 rules including regular expressions, statistical analysis and custom services
-such as URL black lists. Each message is analysed by Rspamd and given a `spam score`. 
+such as URL black lists. Each message is analysed by Rspamd and given a `spam score`.
 
-According to this spam score and the user's settings Rspamd recommends an action for
+According to this spam score and the user's settings, Rspamd recommends an action for
 the MTA to apply to the message, for example, to pass, reject or add a header.
-Rspamd is designed to process hundreds of messages per second simultaneously and has a number of 
-features available.
+Rspamd is designed to process hundreds of messages per second simultaneously, and sports a number of
+useful features.
 
```

Comment:
What's the purpose of this change? I have never seen the word 'sports' used in this context. Another change is also meaningless to my opinion.